Testing/VelaptorTesting/Program.cs

```diff
@@ -14,20 +15,54 @@ namespace VelaptorTesting
     /// </summary>
     public static class Program
     {
+        private const string ShowDebugConsole = "--show-debug-console";
+        private const string True = "true";
         private static IWindow? window;
         private static MainWindow? gameWindow;
 
-        public static void Main()
+        public static void Main(string[] args)
```

Comment:
Implement additional functionality.

- [ ] Implement a message that the application has started.
- [ ] Add some help text explaining the commands.
- [ ] Add another command to show help